configure.ac

```diff
@@ -17,7 +17,7 @@ AC_CONFIG_FILES([Makefile
 
 # Check for pcap
 AC_CHECK_LIB([pcap],[pcap_create])
-AC_CHECK_HEADERS([pcap/pcap.h])
+AC_CHECK_HEADERS([pcap/pcap.h],[],AC_MSG_ERROR([You need to have libpcap-dev installed]))
```

Comment:
Thanks!

The message is a bit of a debianism though, right? It is called libpcap-devel on Fedora for instance.
Maybe a different wording could be used?

"You need to have libpcap development package installed" maybe?